### PR TITLE
Updated script for Flutter v1.7.8+hotfix.3

### PR DESCRIPTION
**Pull request by akassharjun**
Please check if the SHA256 sum is correct.

### CODE REVIEW

File: Casks/flutter.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 cask 'flutter' do
-  version "1.5.4-hotfix.2"
-  sha256 "0b4cacbe84fd68cc9cfe213a90ddcdba558e45c19be30c6173c07a318be34014"
+  version "v1.7.8+hotfix.3"
```
Comment (fbsb) [inline Casks/flutter.rb]: Could you please remove the `v` prefix from the version? It's already included in the URL and is not needed here.

File: Casks/flutter.rb [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+  sha256 "800db14570c5dd9e4e0d3b403ebd9579cd09226c004239a880470eab438f0872"
```
Comment (fbsb) [inline Casks/flutter.rb]: The correct sum is `1bd5d165c79efebfba39e4602d6e07e74b40798630eb7173caedf0f664c53035`. Could you please update that aswell?

**Review by fbsb — CHANGES_REQUESTED** [review]

**Comment by fbsb** [comment]
Also you need to rebase to resolve the conflicts

**Comment by akassharjun** [comment]
Done

**Review by fbsb — APPROVED** [review]

**Comment by fbsb** [comment]
Great! Thank you!

**Pull request merged**